src/client/app/containers/LineChartContainer.ts

```diff
@@ -9,6 +9,8 @@ import getGraphColor from '../utils/getGraphColor';
 import { State } from '../types/redux/state';
 import PlotlyChart, { IPlotlyChartProps } from 'react-plotlyjs-ts';
 import {TimeInterval} from '../../../common/TimeInterval';
+import * as es from 'plotly.js/lib/locales/es';
+import * as fr from 'plotly.js/lib/locales/fr';
```

Comment:
The changes for all graphic types are the same (line, bar, compare, map). Thus, I wonder if an export/import could be created so it is all kept in one place so that future language additions would be easier to make. Not sure if it will work with all the items but something to look into.